src/components/CourseTable/index.jsx

```diff
@@ -108,12 +108,12 @@ class CourseTable extends React.Component {
 
   isOrgWhitelisted() {
     const userOrgs = this.props.publisherUserInfo.organizations;
-    if (!orgWhitelist || (orgWhitelist && orgWhitelist.length === 0)) {
-      // No Whitelist specified allow all orgs
+    if (!orgBlacklist || (orgBlacklist && orgBlacklist.length === 0)) {
+      // No Blacklist specified allow all orgs
       return true;
     }
     for (let i = 0; i < userOrgs.length; i += 1) {
```

Comment:
I think the following would be better:

`return userOrgs.any(org => !orgBlacklist.includes(org.key))`

something like this